### ISSUE TITLE
Failed to apply chaos: rpc error: code = Unknown desc = controller is not supported

### ISSUE THREAD

**Issue by seriousgong** [chaos-mesh/chaos-mesh#2652]
## Bug Report

**What version of Kubernetes are you using?**

1.14.3
**What version of Chaos Mesh are you using?**

2.1.0 made by myself
**What did you do?**

add  stresschaos
**What did you see instead?**
![image](https://user-images.githubusercontent.com/56463479/146206351-58fee1d8-7d99-499d-8056-9bcfd57a4efa.png)

**Output of chaosctl**

yaml
![image](https://user-images.githubusercontent.com/56463479/146206467-e73d13dc-f23d-4df7-b5e4-aaf3b18bd356.png)

**Comment by STRRL** [comment]
Hi @ChanKunggc , are you using cgroup v2? Currently StressChaos does not work well with cgroupv2. :sob:

**Comment by ChanKunggc** [comment]
> 



> Hi @ChanKunggc , are you using cgroup v2? Currently StressChaos does not work well with cgroupv2. sob

Hi @STRRL , actually we use  blkio  with cgroupv2 ,but  delete  this line https://github.com/chaos-mesh/chaos-mesh/blob/802bf850e3e852b2aecc88400f9b5c66cfab9e9c/pkg/chaosdaemon/cgroups/utils.go#L41
it work well , so i think that is doesn't matter.

**Comment by igcherkaev** [comment]
Seeing the same on Kubernetes v1.21.9 and chaos mesh v2.1.3. Does it mean we have to build a custom chaosdaemon image with that line removed?

**Comment by STRRL** [comment]
> Seeing the same on Kubernetes v1.21.9 and chaos mesh v2.1.3. Does it mean we have to build a custom chaosdaemon image with that line removed?

I think it could not works on your situation, because it seems he only enable block io cgroup v2 controlelr.

**Comment by igcherkaev** [comment]
@STRRL thing is, I didn't explicitly enable cgroup v2, and my k8s clusters are like 4 years old, being upgraded in place over time with kubelet configuration fixed to use cgroupfs, containerd is not set to use systemd's cgroup, etc (everything that the docs say to do to migrate to cgroup v2 I did not do), but somehow inside containers it looks like it's using cgroup v2 (at least based on the mount type and items under that mount). Btw, it changed when we upgraded to Kubernetes 1.21, we had a few apps running in k8s that relied on certain items there and they got changed, even though we didn't ask k8s to switch to cgroup v2. 

Anyway, it seems to be inevitable and cgroup v1 will be gone eventually, so my question is: anyone working on this issue for chaos mesh? Is it on the road map anywhere? Maybe there are other ways to simulate cpu usage without using cgroup? Not really sure where the problem is exactly. What is cgroup used for when we want to simulate CPU usage by containers?
